api/storage/app/lighthouse-schema.graphql

```diff
@@ -602,6 +623,7 @@ type User {
   locationExemptions: String
   acceptedOperationalRequirements: [LocalizedOperationalRequirement]
   positionDuration: [PositionDuration]
+  employeeProfile: EmployeeProfile
```

Comment:
The [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12001#:~:text=type%20User%20%7B%0A%20%20type%20employeeProfile%3A%20EmployeeProfile%20%40self%20%40canResolved(ability%3A%20%22view%22)%0A%7D) states: `type employeeProfile: EmployeeProfile @self @canResolved(ability: "view")`.

Reply:
Same as the last one, this is a different model than `User` so `@self` does not make sense here. Unless I just do not understand the `@self` directive 😅

Reply:
I agree this field is about relationship between EmployeeProfile and UserPublicProfile, which are also different types. @self is not needed in here